src/drt/test/BUILD

```diff
@@ -0,0 +1,104 @@
+load("//test:regression.bzl", "regression_test")
+
+# From CMakeLists.txt or_integration_tests(TESTS
+COMPULSORY_TESTS = [
+    "drc_test",
+    "ispd18_sample",
+    "ispd18_sample_incr",
+    "ndr_vias1",
+    "ndr_vias2",
+    "ndr_vias3",
+    "obstruction",
+    "single_step",
+    "ta_ap_aligned",
+    "ta_pin_aligned",
+    "top_level_term",
+    "top_level_term2",
+]
+
+# Disabled in CMakeLists.txt
+MANUAL_TESTS = [
+    "drt_man_tcl_check",
+    "drt_readme_msgs_check",
+]
+
+ALL_TESTS = COMPULSORY_TESTS + MANUAL_TESTS
+
+filegroup(
+    name = "regression_resources",
+    # Dependencies could be specified more narrowly per test case,
+    # but at least it is not a glob of everything and there are
+    # per test glob patterns below.
+    srcs = [
+        ":Nangate45/Nangate45_stdcell.lef",
```

Comment:
if these are non-generated, I'd not put the `:` in front.

Reply:
fixed.